app/views/directives/lifecycle-hook.html

```diff
@@ -1,133 +1,59 @@
-<ng-form name="editForm">
-  <div ng-switch="type">
-    <div class="help-block" ng-switch-when="pre">Pre hooks execute before the deployment begins.</div>
-    <div class="help-block" ng-switch-when="mid">Mid hooks execute after the previous deployment is scaled down to zero and before the first pod of the new deployment is created.</div>
-    <div class="help-block" ng-switch-when="post">Post hooks execute after the deployment strategy completes.</div>
+<h4>
+  {{type | upperFirst}} Hook
+</h4>
+<span ng-switch="type">
+  <small class="text-muted" ng-switch-when="pre">Pre hooks execute before the deployment begins.</small>
+  <small class="text-muted" ng-switch-when="mid">Mid hooks execute after the previous deployment is scaled down to zero and before the first pod of the new deployment is created.</small>
+  <small class="text-muted" ng-switch-when="post">Post hooks execute after the deployment strategy completes.</small>
```

Comment:
Instead of `small`, let's use the `help-block` style. (Margin looked off in the screenshot, but I think `help-block` will fix that.)